app/models/application_info.rb

```diff
@@ -0,0 +1,17 @@
+class ApplicationInfo < ApplicationRecord
+  attribute :apple_app_id
```

Comment:
`attributes` used for overriding attribute type
https://apidock.com/rails/ActiveRecord/Attributes/ClassMethods/attribute

for model annotation better to use gem `annotate`
https://github.com/ctran/annotate_models